### ISSUE TITLE
Digitally signed error in trying to execute gulp or yarn

### ISSUE THREAD

**Issue by EPTamminga** [nvisionative/nvQuickTheme#289]
Installed NVQT on a fresh W11 install. Followed all the steps. I get the following error when trying Yarn version:
```
File C:\Users\ept\AppData\Roaming\npm\yarn.ps1 cannot be loaded. 
The file C:\Users\ept\AppData\Roaming\npm\yarn.ps1 is not digitally signed. 
You cannot run this script on the current system.
```
Similar errors with Gulp
```
C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift> gulp build
gulp : File C:\Users\ept\AppData\Roaming\npm\gulp.ps1 cannot be loaded. 
The file C:\Users\ept\AppData\Roaming\npm\gulp.ps1 is not digitally signed. 
You cannot run this script on the current system.
```
I am trying to find PS security levels. My current output:
```
PS C:\Users\ept\Downloads> Get-ExecutionPolicy -list

        Scope ExecutionPolicy
        ----- ---------------
MachinePolicy       Undefined
   UserPolicy       Undefined
      Process       Undefined
  CurrentUser       Undefined
 LocalMachine       AllSigned 
```
Any suggestions?

**Comment by EPTamminga** [comment]
@bdukes That resolved the security issue, but now there are build errors:

```
PS C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift> gulp build
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './lib/map'
Require stack:
- C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\now-and-later\index.js
- C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\bach\lib\series.js
- C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\bach\index.js
- C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\undertaker\lib\series.js
- C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\undertaker\index.js
- C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\gulp\index.js
- C:\Users\ept\AppData\Roaming\npm\node_modules\gulp\node_modules\gulp-cli\lib\versioned\^4.0.0\index.js
- C:\Users\ept\AppData\Roaming\npm\node_modules\gulp\node_modules\gulp-cli\index.js
- C:\Users\ept\AppData\Roaming\npm\node_modules\gulp\bin\gulp.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\DDrive\Develop\Toegift\Website\Portals\_default\Skins\Toegift\node_modules\now-and-later\index.js:4:8)    
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\DDrive\\Develop\\Toegift\\Website\\Portals\\_default\\Skins\\Toegift\\node_modules\\now-and-later\\index.js',
    'C:\\DDrive\\Develop\\Toegift\\Website\\Portals\\_default\\Skins\\Toegift\\node_modules\\bach\\lib\\series.js',
    'C:\\DDrive\\Develop\\Toegift\\Website\\Portals\\_default\\Skins\\Toegift\\node_modules\\bach\\index.js',
    'C:\\DDrive\\Develop\\Toegift\\Website\\Portals\\_default\\Skins\\Toegift\\node_modules\\undertaker\\lib\\series.js',
    'C:\\DDrive\\Develop\\Toegift\\Website\\Portals\\_default\\Skins\\Toegift\\node_modules\\undertaker\\index.js',
    'C:\\DDrive\\Develop\\Toegift\\Website\\Portals\\_default\\Skins\\Toegift\\node_modules\\gulp\\index.js',
    'C:\\Users\\ept\\AppData\\Roaming\\npm\\node_modules\\gulp\\node_modules\\gulp-cli\\lib\\versioned\\^4.0.0\\index.js',
    'C:\\Users\\ept\\AppData\\Roaming\\npm\\node_modules\\gulp\\node_modules\\gulp-cli\\index.js',
    'C:\\Users\\ept\\AppData\\Roaming\\npm\\node_modules\\gulp\\bin\\gulp.js'
  ]
}

```
Any new suggestions?

**Comment by EPTamminga** [comment]
I took a backup copy of the node_modules folder that I saved earlier and copied that over the Skin node-modules folder. The theme now builds.
Other changes that I had on my newly installed PC: VS2022 i.s.o. VS2019 (I think that created a mismatch with chocolatey PS install in the Node install). Not sure if that created the problem.
Pfff.. It's hard to stay up to date with tool versions, especially if there are a lot that depend on each other.
